src/components/mod-queue-button/mod-queue-button.module.css

```
@@ -0,0 +1,23 @@
+   ModQueueBoardCount styling, which shares the same visual; kept here so the
+   button stays self-contained and does not import the route's stylesheet. */
+.modQueueButtonCount {
+  font-weight: bold;
+}
+
+/* Number blinking, changing color to red */
+.modQueueButtonCountAlert {
+  animation: blinkColor 2s infinite;
+}
+
+@keyframes blinkColor {
```

Stylelint (17.13.0)

[error] Expected keyframe name "blinkColor" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)